src/components/renderer/form-record-list.vue

```diff
@@ -397,44 +407,51 @@ export default {
       this.$emit('input', data);
     },
     showAddForm() {
-      const uniqueId = Math.random().toString(36).substring(2) + Date.now().toString(36);
-      this.$set(this.addItem, 'row_id', uniqueId);
-      this.setUploadDataNamePrefix();
+      this.validateModal = false;
+      // Validate the form is set up correctly
       if (!this.form) {
         this.$refs.infoModal.show();
         return;
       }
+      // Reset addItem to an empty object and with a new row_id
+      const uniqueId = Math.random().toString(36).substring(2) + Date.now().toString(36);
+      this.addItem = {
+        row_id: uniqueId
+      };
+      // Rebuild the addItem screen to initialize the form (defaultValues and variables)
+      this.addFormInstanceKey = uniqueId;
+      this.setUploadDataNamePrefix();
+
       // Open form
       this.$refs.addModal.show();
-
-      // eslint-disable-next-line no-unused-vars
-      let {_parent, ...result} = this.addItem;
-      this.initFormValues = _.cloneDeep(result);
     },
-    handleOk(bvModalEvt) {
+    handleHidden() {
+      if (!this.validateModal) {
+        this.cancelValidations();
```

Comment:
This will enable the Submit button no matter if the form is still invalid

https://user-images.githubusercontent.com/8028650/205065151-dc57746e-b979-4e31-b46c-05935fe2b409.mp4

Reply:
Maybe you could instead try to refresh the validation, with something like:
```
const screen = findScreenOwnerOfRecordList();
const mainScreen=screen.getRootScreen();
this.validate(mainScreen);
```